core/src/klv.c

```diff
@@ -207,6 +212,9 @@ double get_leave_value(KLV *klv, Rack *leave) {
   if (leave->empty) {
     return 0.0;
   }
+  if (klv == NULL) {
```

Comment:
When do we expect this to be true?

Reply:
for situations where we don't have leaves for a specific lexicon